src/libsyntax_pos/symbol.rs

```diff
@@ -100,8 +100,127 @@ symbols! {
-        unstable, rustc_const_unstable,
+        masked,
+        spotlight,
+        alias,
+        keyword,
+        include,
+        simd,
+        align,
+        warn,
+        allow,
+        forbid,
+        deny,
+        macro_use,
+        macro_export,
+        plugin_registrar,
+        cfg,
+        cfg_attr,
+        main,
+        start,
+        repr,
+        path,
+        automatically_derived,
+        no_link,
+        derive,
+        should_panic,
+        ignore,
+        no_implicit_prelude,
+        reexport_test_harness_main,
+        link_args,
+        macro_escape,
+        structural_match,
+        non_exhaustive,
+        marker,
+        plugin,
+        no_std,
+        no_core,
+        lang,
+        linkage,
+        thread_local,
+        rustc_on_unimplemented,
+        rustc_const_unstable,
+        global_allocator,
+        default_lib_allocator,
+        needs_allocator,
+        panic_runtime,
+        needs_panic_runtime,
+        rustc_outlives,
+        rustc_variance,
+        rustc_layout,
+        rustc_layout_scalar_valid_range_start,
+        rustc_layout_scalar_valid_range_end,
+        rustc_regions,
+        rustc_error,
+        rustc_dump_user_substs,
+        rustc_if_this_changed,
+        rustc_then_this_would_need,
+        rustc_dirty,
+        rustc_clean,
+        rustc_partition_reused,
+        rustc_partition_codegened,
+        rustc_expected_cgu_reuse,
+        rustc_synthetic,
+        rustc_symbol_name,
+        rustc_def_path,
+        rustc_mir,
+        rustc_inherit_overflow_checks,
+        rustc_dump_program_clauses,
+        rustc_test_marker,
+        rustc_transparent_macro,
+        compiler_builtins,
+        sanitizer_runtime,
+        profiler_runtime,
+        allow_internal_unstable,
+        allow_internal_unsafe,
+        fundamental,
+        proc_macro_derive,
+        rustc_copy_clone_marker,
+        doc,
+        cold,
+        naked,
+        ffi_returns_twice,
+        target_feature,
+        export_name,
+        inline,
+        link,
+        link_name,
+        link_section,
+        no_builtins,
+        no_mangle,
+        no_debug,
+        omit_gdb_pretty_printer_section,
+        unsafe_destructor_blind_to_params,
+        may_dangle,
+        unwind,
+        used,
+        prelude_import,
+        rustc_deprecated,
+        must_use,
+        stable,
+        unstable,
+        deprecated,
+        rustc_paren_sugar,
+        windows_subsystem,
+        proc_macro_attribute,
+        proc_macro,
+        rustc_proc_macro_decls,
+        allow_fail,
+        rustc_std_internal_symbol,
+        rustc_conversion_suggestion,
+        rustc_args_required_const,
+        panic_handler,
+        alloc_error_handler,
+        optimize,
+        crate_name,
+        crate_type,
+        crate_id,
+        feature,
+        no_start,
+        no_main,
+        recursion_limit,
+        type_length_limit,
+        test_runner,
```

Comment:
Nit: could you sort this list alphabetically.

Reply:
Why?

Reply:
For aesthetic reasons :)